client-course-schedulizer/src/components/reuseables/GridItem/GridItemTextField/GridItemTextField.tsx

```diff
@@ -0,0 +1,24 @@
+import { Grid, StandardTextFieldProps, TextField } from "@material-ui/core";
+import React from "react";
+import { useForm } from "react-hook-form";
+import "./GridItemTextField.scss";
+
+export const GridItemTextField = ({
+  label,
+  register,
+  textFieldProps,
+}: {
+  label: string;
+  register: ReturnType<typeof useForm>["register"];
+  textFieldProps?: StandardTextFieldProps;
+}) => {
+  return (
+    <Grid item xs>
+      <TextField inputRef={register} label={label} name={label.toLowerCase()} {...textFieldProps} />
+    </Grid>
+  );
+};
+
+GridItemTextField.defaultProps = {
+  textFieldProps: {},
+};
```

Comment:
I'll make an interface, it's probably better practice

Reply:
Actually, I realized that there's no need for an interface since StandardTextFieldProps is already imported. I'll just set the default value to undefined rather than an empty object